jest-playwright.config.js

```diff
@@ -0,0 +1,4 @@
+module.exports = {
+  USE_NEW_API: true,
```

Comment:
why no camel case here?

Reply:
@mxschmitt I think that snake style will show that it's experimental feature